.goreleaser.yaml

```diff
@@ -53,16 +53,16 @@ changelog:
     - go mod tidy
   groups:
     - title: Dependency updates
-      regexp: "^.*(feat|fix)\\(deps\\)*:+.*$"
```

Comment:
actually I don't think we really need the `^.*` prefix anyway

Reply:
Yer I thought that too, which is why I also added the comment to clarify that it's not just checking against the commit title but also the abbreviated commit reference too.

Reply:
If we did remove the `abbrev-commit` prefix then it might match the pattern anywhere in the title vs anchored to the stat, which is not the intention.

Reply:
ahh, right! thanks